src/types/redis_hash.cc

```diff
@@ -355,4 +383,36 @@ rocksdb::Status Hash::Scan(const Slice &user_key, const std::string &cursor, uin
   return SubKeyScanner::Scan(kRedisHash, user_key, cursor, limit, field_prefix, fields, values);
 }
 
+Status Hash::ParseRangeLexSpec(const std::string &min, const std::string &max, CommonRangeLexSpec *spec) {
```

Comment:
Could you move the function to range_spec.cc, so it can be reused in zset and hash?

Reply:
OK, I am very sorry for missing this function.